lua/due_nvim.lua

```diff
@@ -1,28 +1,57 @@
 local M = {}
 _VT_NS = vim.api.nvim_create_namespace("lsp_signature")
 
+
 local function parseDue(due)
   local year = 31556926
   local month = 2629743
   local week = 604800
   local day = 86400
+  local hour = 3600
+  local minute = 60
   local res = ''
+
   if due >= year then
     res = res .. math.floor(due / year) .. 'y '
     due = due % year
   end
+
   if due >= month then
-    res = res .. math.floor(due / month) .. 'm '
+    res = res .. math.floor(due / month) .. 'mo '
```

Comment:
nah, 'mo' looks weird, wouldn't it be better just to use 'min' for minutes?

Reply:
Good plan!